### PR TITLE
Added a few profile changes, added email validation for user route, and fixed validation alias so it works like the rest of the /data/ files

### PR DESCRIPTION
**Pull request by zzwerling**

### CODE REVIEW

File: api/routes/ratings.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+router.post('/retrieve', async function(req, res){
+    let userId = req.body.userId;
+    try{
+        let ratings = await ratingsData.getAllRatingsByUserId(new ObjectId(userId));
+        console.log(ratings);
+        res.send(ratings);
+
+    }catch(error){
+        console.log(error);
+        res.send(error);
```
Comment (JordanH821) [inline api/routes/ratings.js]: set the HTTP statuts code. `res.status(ERROR_CODE).send(error)`

File: api/routes/users.js [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
-        res.send(error);
+    if (!await validation.isGoodEmail(email)){
+        res.status(400).send("Invalid email!");
+    }else{
+        try{
+            const user = await usersData.getUserByEmail(email);
+            res.send(user);
+    
+        }catch(error){
+            res.send(error);
```
Comment (JordanH821) [inline api/routes/users.js]: set status, probably 500 in the catch

File: frontend/src/components/Profile.js [hunk 3]
```diff
@@ -2,39 +2,75 @@ import React, { useEffect, useState } from 'react';
...
+    const [ratingData, setRatingData] = useState(undefined);
+    const [ratingIdData, setRatingIdData] = useState(undefined);
     const loaded = false;
-    const email = 'bsanders@gmail.com';
-    const url = 'http://localhost:5000/users/profile';
-
-    const requestOptions = {
+    const email = 'donotpassgo@gmail.com';
+    const infoUrl = 'http://localhost:5000/users/profile';
+    const ratingsUrl = 'http://localhost:5000/ratings/retrieve';
```
Comment (JordanH821) [inline frontend/src/components/Profile.js:13]: update to include the Google id token

**Review by JordanH821 — CHANGES_REQUESTED** [review]

File: api/routes/ratings.js (hunk 1 above)
Comment (zzwerling) [reply api/routes/ratings.js]: fixed

File: api/routes/users.js (hunk 2 above)
Comment (zzwerling) [reply api/routes/users.js]: fixed

File: frontend/src/components/Profile.js (hunk 3 above)
Comment (zzwerling) [reply frontend/src/components/Profile.js:13]: already fixed

**Review by JordanH821 — APPROVED** [review]

**Pull request merged**